rocketpy/mathutils/function.py

```diff
@@ -39,14 +39,9 @@ def __init__(
 
         Parameters
         ----------
-        source : function, scalar, ndarray, string
-            The actual function. If type is function, it will be called for
-            evaluation. If type is int or float, it will be treated as a
-            constant function. If ndarray, its points will be used for
-            interpolation. An ndarray should be as [(x0, y0, z0), (x1, y1, z1),
-            (x2, y2, z2), ...] where x0 and y0 are inputs and z0 is output. If
-            string, imports file named by the string and treats it as csv.
-            The file is converted into ndarray and should not have headers.
+        source : function, scalar, ndarray, string, Function
+            The data source to be used for the function. See the ``set_source``
+            method's documentation for more details.
```

Comment:
Having a description of what source is here is much more useful than it just referencing another method. I know it makes for repeated documentation, but I would argument that sometimes repetition in documentation can be good, and this is one of those cases.

Honestly, from time to time I have to check the description of `source` and having to look at a method's argument to understand what I need to do to initialize the object instead of just looking at `__init__` would be annoying and could be confusing for people not used to python library documentation styles

The current description in `set_source` is really good, I'd just copy and paste it here

Reply:
I don't think repeating docstrings is a good idea. I could agree that having it in the init method is better than in the set_source method.

But I won't say anything if you duplicate this docstring to accomplish with your comment.